modules/cutesv/1.0/config/default.yaml

```diff
@@ -6,17 +6,26 @@ lcr-modules:
             sample_bam: "__UPDATE__"
             sample_bai: "__UPDATE__"
         
-        scratch_subdirectories: []
+        scratch_subdirectories: ["cutesv_working"]
```

Comment:
Could you please leave this blank for the default? It can cause some issues on re-runs of the same module.

Reply:
Fixed!